PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -28,6 +28,13 @@ def miniAOD_customizeCommon(process):
     process.patMuons.computeMiniIso = cms.bool(True)
     process.patMuons.computeMuonMVA = cms.bool(True)
     
+    # EGamma objects from HGCal are not yet in GED
+    # so add companion collections for Phase-II MiniAOD production
+    from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+    from PhysicsTools.PatAlgos.slimming.addHGCalEgammaCollections import addHGCalEgammaCollections
+    if phase2_hgcal.isChosen():
```

Comment:
"isChosen" is deprecated
it should be replaced by relevant era modifiers (toModify, toReplaceWith).

Reply:
Hi @slava77 
Is it kosher to pass `process` to `toModify` ?
e.g.
```python
phase2_hgcal.toModify(process, addHGCalEgammaCollections)
```
The other pattern I see is to put toReplaceWith in the `cff` of these sequences, but I consciously tried to keep all the sequence changes in one file so it can easily be removed when HGCal EGamma are in the particle flow.

Reply:
it is OK to pass ```phase2_hgcal.toModify(process.module, parameter = value)```
if it is desired to have a single point of control to add/remove this,
I can suggest to add phase2_hgcalMultiCl (or similarly descriptive name) and chain it with phase2_hgcal modifier.